### PR TITLE
remove the image_extensions flag (almost) completely

### PR DESCRIPTION
**Pull request by akshay-krishnan**
It was only being used in one_d_sfm loader. 

Now they use use a util to find all images in a directory. 

I also updated the CI benchmark to not pass this flag. It is however still a part of the artifact name, because changing this could break our offline scripts to parse any artifacts. It is also tricky (though very doable) to change the indices of the remaining fields from the "dataset matrix config" used in the CI.

### CODE REVIEW

**Comment by akshay-krishnan** [comment]
my changes from another branch creeped in, lets wait until that is merged

File: gtsfm/bundle/bundle_adjustment.py
Comment (ayushbaid) [inline gtsfm/bundle/bundle_adjustment.py:481]: Where do we use this function?

**Review by ayushbaid — COMMENTED** [review]
I'll review this PR after the other one gets merged

**Review by ayushbaid — APPROVED** [review]
LGTM!

**Pull request merged**
